src/lib/y2network/widgets/interface_name.rb

```diff
@@ -0,0 +1,71 @@
+require "yast"
+require "cwm/common_widgets"
+
+Yast.import "NetworkInterfaces"
+Yast.import "LanItems"
+
+module Y2Network
+  module Widgets
+    class InterfaceName < CWM::ComboBox
+      def initialize(settings)
+        textdomain "network"
+
+        @settings = settings
+      end
+
+      def label
+        _("&Configuration Name")
```

Comment:
Currently it is the interface name and not the configuration one.

NP: It would be nice the widget to be inside a Frame just for be consistent with the rest of the layout.

Reply:
Well, not necessary, some are inside frame and some not. I think we probably need to unify it.

about interface versus configuration name. I just copy label from previous one. I plan to revisit all texts if it makes sense later when we can easily change it in each widget.

Reply:
Ok, then. Maybe being just above the "Device Activation", using "Connection" and then "Device" instead of using same term was somehow strange.

But we were already discussing about use a matching rule for matching the connection configuration with an interface so.. let it as it is.

Reply:
:+1: about revisit all the texts. I think you are doing good improvements with each CWM object that you are currently rewritten.

Reply:
BTW: I was thinking that we could use always "Device Name", and when it was named with udev, just show that with the change button and maybe show the description the match rule key ("bus_id" or "mac address")... but that is just and idea